src/scripts/ccget

```diff
@@ -1,14 +1,9 @@
-#!/usr/bin/env python3
```

Comment:
That's correct, thanks. I think we had python3 explicitly in order to force Py3 for systems that had both. I don't think it matters much, though I can re-add it if you think it's worthwhile.